files/image_config/warmboot-finalizer/finalize-warmboot.sh

```diff
@@ -18,6 +28,20 @@ function debug()
 }
 
 
+function filter_component_list()
+{
+    CP_LIST=${!RECONCILE_COMPONENTS[@]}
+    COMPONENT_LIST=""
+    for cp in ${CP_LIST}; do
+        service=${RECONCILE_COMPONENTS[${cp}]}
+	status=$(sonic-db-cli CONFIG_DB HGET "FEATURE|${service}" state)
```

Comment:
The indentation looks a little off.

Reply:
yeah, editing on the target dut. vim setting is wrong there.